modules/openxr/extensions/openxr_fb_passthrough_extension_wrapper.cpp

```diff
@@ -0,0 +1,418 @@
+
+	result = openxr_api->get_instance_proc_addr("xrPassthroughLayerPauseFB", (PFN_xrVoidFunction *)&xrPassthroughLayerPauseFB_ptr);
+	ERR_FAIL_COND_V(XR_FAILED(result), false);
+
+	result = openxr_api->get_instance_proc_addr("xrPassthroughLayerResumeFB", (PFN_xrVoidFunction *)&xrPassthroughLayerResumeFB_ptr);
+	ERR_FAIL_COND_V(XR_FAILED(result), false);
+
+	result = openxr_api->get_instance_proc_addr("xrPassthroughLayerSetStyleFB", (PFN_xrVoidFunction *)&xrPassthroughLayerSetStyleFB_ptr);
+	ERR_FAIL_COND_V(XR_FAILED(result), false);
+
+	result = openxr_api->get_instance_proc_addr("xrCreateGeometryInstanceFB", (PFN_xrVoidFunction *)&xrCreateGeometryInstanceFB_ptr);
+	ERR_FAIL_COND_V(XR_FAILED(result), false);
+
+	result = openxr_api->get_instance_proc_addr("xrDestroyGeometryInstanceFB", (PFN_xrVoidFunction *)&xrDestroyGeometryInstanceFB_ptr);
+	ERR_FAIL_COND_V(XR_FAILED(result), false);
+
+	result = openxr_api->get_instance_proc_addr("xrGeometryInstanceSetTransformFB", (PFN_xrVoidFunction *)&xrGeometryInstanceSetTransformFB_ptr);
+	ERR_FAIL_COND_V(XR_FAILED(result), false);
+
+	return true;
+}
+
+bool OpenXRFbPassthroughExtensionWrapper::initialize_fb_triangle_mesh_extension(const XrInstance p_instance) {
+	XrResult result;
+	ERR_FAIL_NULL_V(openxr_api, false);
+
+	result = openxr_api->get_instance_proc_addr("xrCreateTriangleMeshFB", (PFN_xrVoidFunction *)&xrCreateTriangleMeshFB_ptr);
+	ERR_FAIL_COND_V(XR_FAILED(result), false);
+
+	result = openxr_api->get_instance_proc_addr("xrDestroyTriangleMeshFB", (PFN_xrVoidFunction *)&xrDestroyTriangleMeshFB_ptr);
+	ERR_FAIL_COND_V(XR_FAILED(result), false);
+
+	result = openxr_api->get_instance_proc_addr("xrTriangleMeshGetVertexBufferFB", (PFN_xrVoidFunction *)&xrTriangleMeshGetVertexBufferFB_ptr);
+	ERR_FAIL_COND_V(XR_FAILED(result), false);
+
+	result = openxr_api->get_instance_proc_addr("xrTriangleMeshGetIndexBufferFB", (PFN_xrVoidFunction *)&xrTriangleMeshGetIndexBufferFB_ptr);
+	ERR_FAIL_COND_V(XR_FAILED(result), false);
+
+	result = openxr_api->get_instance_proc_addr("xrTriangleMeshBeginUpdateFB", (PFN_xrVoidFunction *)&xrTriangleMeshBeginUpdateFB_ptr);
+	ERR_FAIL_COND_V(XR_FAILED(result), false);
+
+	result = openxr_api->get_instance_proc_addr("xrTriangleMeshEndUpdateFB", (PFN_xrVoidFunction *)&xrTriangleMeshEndUpdateFB_ptr);
+	ERR_FAIL_COND_V(XR_FAILED(result), false);
+
+	result = openxr_api->get_instance_proc_addr("xrTriangleMeshBeginVertexBufferUpdateFB", (PFN_xrVoidFunction *)&xrTriangleMeshBeginVertexBufferUpdateFB_ptr);
+	ERR_FAIL_COND_V(XR_FAILED(result), false);
+
+	result = openxr_api->get_instance_proc_addr("xrTriangleMeshEndVertexBufferUpdateFB", (PFN_xrVoidFunction *)&xrTriangleMeshEndVertexBufferUpdateFB_ptr);
+	ERR_FAIL_COND_V(XR_FAILED(result), false);
+
+	return true;
+}
+
+XrResult OpenXRFbPassthroughExtensionWrapper::xrCreatePassthroughFB(XrSession p_session, const XrPassthroughCreateInfoFB *p_createInfo, XrPassthroughFB *p_outPassthrough) {
```

Comment:
Food for thoughts.. we were using a macro in Godot 3 to quickly add these methods since they follow the same pattern.
Might be worth considering using the same approach here as well.

Reply:
I think it is worth re-introducing those for sure

Reply:
Good point, we will take a look tomorrow.